cloud/pkg/cloudhub/servers/httpserver/certutil.go

```diff
@@ -114,7 +116,7 @@ func NewCertFromCa(cfg *certutil.Config, caCert *x509.Certificate, serverKey cry
 		IPAddresses:  cfg.AltNames.IPs,
 		SerialNumber: serial,
 		NotBefore:    time.Now().UTC(),
-		NotAfter:     time.Now().Add(time.Hour * 24 * 365 * 100),
+		NotAfter:     time.Now().Add(time.Hour * 24 * time.Duration(t)),
```

Comment:
Here do we need to count how many days in a year, or just hardcode 365 days?

Reply:
Oh, i see this config option `EdgeCertValidityPeriod` while not see where to call it, is it normal?

Reply:
> Here do we need to count how many days in a year, or just hardcode 365 days?

We don't count the days in a year. The certificate of CloudCore remains valid for 36500 days. While the certificate of EdgeCore can be manually set in `EdgeCertValidityPeriod` .

Reply:
> Oh, i see this config option `EdgeCertValidityPeriod` while not see where to call it, is it normal?

![image](https://user-images.githubusercontent.com/38457133/85520605-ba19db00-b635-11ea-9006-3411618f4155.png)
Here we call it to set the valid period of EdgeCore certificate.

Reply:
Oh i am confused with `validalityPeriod` and `EdgeCertValidityPeriod`.